x-pack/plugins/canvas/common/lib/url.js

```diff
@@ -4,9 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { isValid as isValidDataUrl } from '../../common/lib/dataurl';
-import { isValid as isValidHttpUrl } from '../../common/lib/httpurl';
+import { isValidDataUrl } from '../../common/lib/dataurl';
+import { isValidHttpUrl } from '../../common/lib/httpurl';
 
-export function isValid(url) {
+export function isValidUrl(url) {
```

Comment:
Why did you rename this export? Was it required for some reason?

Reply:
same as above